docs/installation/SUMMA_on_OS_X.md

```diff
@@ -62,4 +65,63 @@ In the following I will assume that you don't have a Fortran compiler or NetCDF
 
  1. Now obtain the SUMMA source code from the [SUMMA source code repository](https://github.com/NCAR/summa). You may just want to download the latest tagged release. Unless you are planning to contribute to the source code, there is no need to clone or fork the repository.
 
- 1. Untar or unzip the archive, then go to the `summa/build` directory and follow the instructions in the [SUMMA installation](SUMMA_installation.md) page. If you are using MacPorts, the `FC_ENV` can be set to `gfortran-6-macports`.
+ 2. Untar or unzip the archive, then go to the `summa/build` directory and follow the instructions in the [SUMMA installation](SUMMA_installation.md) page. If you are using MacPorts, the `FC_ENV` can be set to `gfortran-6-macports`.
+
+ ## Instructions with homebrew
+
+ These instructions deviate from the standard SUMMA on OS X installation instructions in that Homebrew is used as an alternative
+ to MacPorts as a package installer. This is largely due to Catalina changes to a users /opt/ folder.
+
+ These instructions assume that you've already cloned the SUMMA repository.
+ If you haven't, follow the initial [SUMMA Installation Instructions](https://summa.readthedocs.io/en/latest/installation/SUMMA_installation/)
+
+ ### Step 1: Installation of Homebrew
```

Comment:
Can you ensure consistent formatting of steps for both the MacPorts (currently numbered list) and Homebrew (currently sub-headings) sections?

Reply:
Added sub-headings to MacPorts part, and numbered list to Homebrew part